guide/ops/locations/_AWS.md

```diff
@@ -111,3 +125,15 @@ You can then deploy blueprints to the subnet, allowing VPC hardware instance typ
 by specifying the subnet ID as the `networkName` in your YAML blueprint.
 This is covered in the previous section, "Using Subnets".
 
+
+### Tidying up after jclouds
+
+Security groups are not always deleted by jclouds. This is due to a limitation in AWS (see
+https://issues.apache.org/jira/browse/JCLOUDS-207). In brief, AWS prevents the security group
+being deleted until there are no VMs using it. However, there is eventual consistency for
+recording which VMs still reference those security groups: after deleting the VM, it can sometimes
+take several minutes before the security group can be deleted. jclouds retries for 3 seconds, but
```

Comment:
Not sure I follow this rewording - I prefer the original. It is AWS's use of eventual consistency that causes the problem (rather than immediate consistency) - hence saying "However" rather than "Whilst there is eventual consistency...".